src/libraries/Common/src/System/Net/Http/aspnetcore/Http2/Hpack/Huffman.cs

```diff
@@ -12,270 +12,532 @@ internal class Huffman
+            0b11111111_11111111_11111001_00000000,
+            0b11111111_11111111_11111011_11000000,
+            0b11111111_11111111_11111011_11100000,
+            0b11111111_11111111_11111001_01000000,
+            0b11111111_11111111_11110001_00000000,
+            0b11111111_11111111_11110110_10000000,
+            0b11111111_11111110_01000000_00000000,
+            0b11111111_11111111_00011000_00000000,
+            0b11111111_11111111_11111001_10000000,
+            0b11111111_11111111_11111100_00000000,
+            0b11111111_11111111_11111100_00100000,
+            0b11111111_11111111_11111001_11000000,
+            0b11111111_11111111_11111100_01000000,
+            0b11111111_11111111_11110010_00000000,
+            0b11111111_11111111_00100000_00000000,
+            0b11111111_11111111_00101000_00000000,
+            0b11111111_11111111_11111010_00000000,
+            0b11111111_11111111_11111010_01000000,
+            0b11111111_11111111_11111111_11010000,
+            0b11111111_11111111_11111100_01100000,
+            0b11111111_11111111_11111100_10000000,
+            0b11111111_11111111_11111100_10100000,
+            0b11111111_11111110_11000000_00000000,
+            0b11111111_11111111_11110011_00000000,
+            0b11111111_11111110_11010000_00000000,
+            0b11111111_11111111_00110000_00000000,
+            0b11111111_11111111_10100100_00000000,
+            0b11111111_11111111_00111000_00000000,
+            0b11111111_11111111_01000000_00000000,
+            0b11111111_11111111_11100110_00000000,
+            0b11111111_11111111_10101000_00000000,
+            0b11111111_11111111_10101100_00000000,
+            0b11111111_11111111_11110111_00000000,
+            0b11111111_11111111_11110111_10000000,
+            0b11111111_11111111_11110100_00000000,
+            0b11111111_11111111_11110101_00000000,
+            0b11111111_11111111_11111010_10000000,
+            0b11111111_11111111_11101000_00000000,
+            0b11111111_11111111_11111010_11000000,
+            0b11111111_11111111_11111100_11000000,
+            0b11111111_11111111_11111011_00000000,
+            0b11111111_11111111_11111011_01000000,
+            0b11111111_11111111_11111100_11100000,
+            0b11111111_11111111_11111101_00000000,
+            0b11111111_11111111_11111101_00100000,
+            0b11111111_11111111_11111101_01000000,
+            0b11111111_11111111_11111101_01100000,
+            0b11111111_11111111_11111111_11100000,
+            0b11111111_11111111_11111101_10000000,
+            0b11111111_11111111_11111101_10100000,
+            0b11111111_11111111_11111101_11000000,
+            0b11111111_11111111_11111101_11100000,
+            0b11111111_11111111_11111110_00000000,
+            0b11111111_11111111_11111011_10000000,
+            0b11111111_11111111_11111111_11111100,
+        };
+
+        private static readonly byte[] s_encodingTableOfLenght = new byte[257]
```

Comment:
```suggestion
        private static ReadOnlySpan<byte> s_encodingTableOfLenght => new byte[257]
```
To use C# compiler optimization for one-byte-sized static data. See [this blog](https://vcsjones.dev/2019/02/01/csharp-readonly-span-bytes-static/) for further infos.

Reply:
This has been moved to another PR